301-web-app-regional-vnet-private-endpoint-sql-storage/azuredeploy.json

```diff
@@ -0,0 +1,458 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located including a trailing '/'"
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/301-web-app-regional-vnet-private-endpoint-sql-storage/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "secureString",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated. Use the defaultValue if the staging location is not secured."
+            },
+            "defaultValue": ""
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "deployment location"
+            }
+        },
+        "webAppName": {
+            "type": "string",
+            "metadata": {

```

Comment:
add a globally unique defaultValue

Reply:
Added a the default value below. 

"defaultValue": "[concat('web-app-', uniqueString(subscription().id, resourceGroup().id))]",